### PR TITLE
feat(feedback): Show the short-id of feedbacks in the list & details

### PR DESCRIPTION
**Pull request by ryan953**
Added in the feedback short-code to the UI. It basically replaces the Project name on the list and details sections of the page:

| Where | Pic | 
| --- | --- |
| List Item | SCR-20231205-ppbk |
| Details header | SCR-20231205-ppda |

Along the way I also trimmed some extra dom nodes and css/styled components away in the `FeedbackListItem` component. It turns out that I was able to make project-name overflow work better, because that bottom row of project + icons is a single grid now. Now a project name will expand to take as much space as possible, but always leave space for any icons that are there. This means the project name can expand under the timestamp, making the grid a little more fluid than before.
SCR-20231205-pqzb

### CODE REVIEW

File: static/app/components/feedback/feedbackItem/feedbackItem.tsx [hunk 1]
```diff
@@ -84,17 +91,33 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
...
             </Flex>
           </Flex>
           <Flex gap={space(1)} align="center" wrap="wrap">
-            <Button
-              title={t('Copy link to this feedback')}
-              size="xs"
-              onClick={copyLink}
-              aria-label={t('Copy Link')}
-              icon={<IconLink />}
-            />
```
Comment (ryan953) [inline static/app/components/feedback/feedbackItem/feedbackItem.tsx:97]: I removed the stand-alone button and put it inside the dropdown, it's copy+pasted from Issues

I think we should still re-vamp the details header here, which will be in a followup.

File: static/app/components/feedback/list/feedbackListItem.tsx [hunk 2]
```diff
@@ -37,22 +37,6 @@ interface Props {
...
-export function FeedbackIcon({
-  tooltipText,
-  icon,
-}: {
-  icon: ReactNode;
-  tooltipText: string;
-}) {
-  return (
-    <StyledTooltip
-      title={<span style={{textTransform: 'capitalize'}}>{tooltipText}</span>}
```
Comment (ryan953) [inline static/app/components/feedback/list/feedbackListItem.tsx:49]: With the recent addition of `getPluginNames`  and the other `get*Names` functions inside issueTrackingSignals, turns out we don't need the textTransform anymore.

File: static/app/components/feedback/list/issueTrackingSignals.tsx [hunk 3]
```diff
@@ -27,20 +33,27 @@ function filterLinkedPlugins(actions: ExternalIssueComponent[]) {
...
+function getIntegrationNames(integrationIssue: IntegrationComponent) {
   return {
     name: integrationIssue.props.configurations[0].provider.name ?? '',
     icon: integrationIssue.key ?? '',
   };
 }
 
+function getAppIntegrationNames(integrationIssue: SentryAppIssueComponent) {
+  return {
+    name: integrationIssue.props.sentryApp.name,
```
Comment (ryan953) [inline static/app/components/feedback/list/issueTrackingSignals.tsx:52]: i'm not sure if this is the right/best name to show. But i think before we had no name for the `SentryAppIssueComponent` case.

**Review by billyvg — APPROVED** [review]

**Comment by jas-kas** [comment]
@ryan953 small tweak but can we say "Copy Feedback URL" (as opposed to Issue, we don't refer to feedbacks that way in our docs and also to avoid confusion with the "Linked Issue")

**Comment by ryan953** [comment]
> @ryan953 small tweak but can we say "Copy Feedback URL" (as opposed to Issue, we don't refer to feedbacks that way in our docs and also to avoid confusion with the "Linked Issue")

@jas-kas fixed! that was some bad copy paste i did

**Pull request merged**